app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -4,11 +4,12 @@ class FormObjectFactory
       class << self
         def form_object_for(step, enrollment)
           klass = form_object_class_map.fetch(step.to_sym) do
-            fail "No form object defined for step #{step}"
+            raise "No form object defined for step #{step}"
           end
           klass.factory(enrollment)
         end
 
+        # rubocop:disable Metrics/MethodLength
         # NB: use NullForm for steps with no html form.
```

Comment:
As the method does nothing but define a hash, I think it's valid to disable the check for this instance. May well be worth refactoring this once all the NullForm assignments become redundant.